ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -126,8 +134,10 @@ class NavigationControllerDemoController: DemoController {
         content.navigationItem.contentScrollView = contractNavigationBarOnScroll ? content.tableView : nil
         content.showsTopAccessoryView = showsTopAccessory
 
-        if style == .custom {
-            content.navigationItem.customNavigationBarColor = CustomGradient.getCustomBackgroundColor(width: view.frame.width)
+        content.navigationItem.customNavigationBarColor = CustomGradient.getCustomBackgroundColor(width: view.frame.width)
```

Comment:
is it safe to remove the style == .custom? how does it not affect other demo scenarios?

Reply:
We only use the customNavigationBarColor when the style is custom anyways, so I wasn't sure exactly why we had the check - as it didn't matter :)